tests/test_ruff_format.py

```diff
@@ -121,3 +127,22 @@ def test_ruff_format_and_sort_imports(workspace):
     )
     got = run_plugin_format(workspace, doc)
     assert want == got
+
+
+def test_ruff_format_via_all(workspace):
```

Comment:
This is somewhat of a fragile thing to test since it's at the mercy of whatever rules ruff might add. The tested text might be reasonably robust, but it also might be best to just leave this out.